src/main/java/org/fundacionjala/coding/movies/andy/Customer.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.movies.andy;
+
+import java.util.ArrayList;
+
+
+class Customer {
+    private String name;
+    private ArrayList<Rental> rentals = new ArrayList<>();
+
+    Customer(final String name) {
+
+        this.name = name;
+    }
+
+    public void addRental(final Rental arg) {
+
+        rentals.add(arg);
+    }
+
+    public String getName() {
+
+        return name;
+    }
+
+    public String statement() {
```

Comment:
This method is doing 3 things. it is printing, and it is calculating the amount and the frequentRenterPoints